tests/utilities/hashingModule.spec.ts

```
@@ -0,0 +1,125 @@
+        "@missinglocal.com",
+        "missing@.com",
+      ];
+
+      invalidEmails.forEach((email) => {
+        expect(() => hashEmail(email)).toThrow("Invalid email format");
+      });
+    });
+
+    it("should throw an error if HASH_PEPPER is missing", () => {
+      const originalPepper = process.env.HASH_PEPPER;
+      delete process.env.HASH_PEPPER;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)